clients/da-vinci-client/src/main/java/com/linkedin/davinci/DaVinciBackend.java

```diff
@@ -110,6 +106,8 @@ public class DaVinciBackend implements Closeable {
   private IngestionBackend ingestionBackend;
   private final AggVersionedStorageEngineStats aggVersionedStorageEngineStats;
   private final boolean useDaVinciSpecificExecutionStatusForError;
+  private final ClientConfig clientConfig;
+  private BlobTransferManager<Void> blobTransferManager;
```

Comment:
Blobs will be file types. Depending on the blob system we use, they have the different type on PUT AP. For instance, it can be nothing for P2P, UUID for some systems, token for another system. That's why I originally wanted to introduce the type to transfer manager.